### ISSUE TITLE
mantl/cert is absent

### ISSUE THREAD

**Issue by dcheperd** [CiscoCloud/mantl#1644]
- Ansible version (`ansible --version`):  ansible 1.9.5
- Python version (`python --version`): ansible 2.7.6
- Git commit hash or branch: upstream/master
- Cloud Environment: OpenStack

Ansible Automation fails because /etc/pki/mantl/cert is absent.

ASK: [certificates | set permissions on certificate files] ******************* 
failed: [mantl-962803299-worker-001] => (item={'path': '/etc/pki/mantl/cert', 'group': 'root', 'mode': '0644'}) => {"failed": true, "item": {"group": "root", "mode": "0644", "path": "/etc/pki/mantl/cert"}, "path": "/etc/pki/mantl/cert", "state": "absent"}
msg: file (/etc/pki/mantl/cert) is absent, cannot continue
ok: [mantl-962803299-worker-001] => (item={'path': '/etc/pki/mantl/key', 'group': 'tls', 'mode': '0640'})

**Comment by stevendborrelli** [comment]
@siddharthist can you work with @dcheperd on this? These changes caused some of his tasks

**Comment by siddharthist** [comment]
Definitely. Can you post your full log @dcheperd? And your modified `roles/certificates/defaults/main.yml` or `roles/certificates/vars/main.yml`, if you have them?

**Comment by siddharthist** [comment]
Ping @dcheperd
